apps/explorer/lib/explorer/counters/address_token_transfers_counter.ex

```diff
@@ -11,7 +11,7 @@ defmodule Explorer.Counters.AddressTokenTransfersCounter do
   @cache_name :address_token_transfers_counter
   @last_update_key "last_update"
 
-  config = Application.get_env(:explorer, __MODULE__)
+  config = Application.compile_env(:explorer, __MODULE__)
```

Comment:
whine: so this is the kind of thing that should be qualified with a keypath

```elixir
Application.compile_env(:explorer, [__MODULE__, :enable_consolidation])
```

would enable the config to change at runtime except for the `:enable_consolidation` key

but this isn't even a protocol why is consolidation a thing?

it literally does nothing

```
  @impl true
  def handle_continue(:ok, %{consolidate?: true} = state) do
    {:noreply, state}
  end

  @impl true
  def handle_continue(:ok, state) do
    {:noreply, state}
  end

  @impl true
  def handle_info(:consolidate, state) do
    {:noreply, state}
  end
```

Reply:
Created a ticket to come back and address it.